components/script/dom/rtcerror.rs

```diff
@@ -28,18 +28,9 @@ pub(crate) struct RTCError {
 }
 
 impl RTCError {
-    fn new_inherited(
-        global: &GlobalScope,
-        init: &RTCErrorInit,
-        message: DOMString,
-        can_gc: CanGc,
-    ) -> RTCError {
+    fn new_inherited(init: &RTCErrorInit, message: DOMString) -> RTCError {
         RTCError {
-            exception: Dom::from_ref(&*DOMException::new(
-                global,
-                DOMErrorName::from(&message).unwrap(),
-                can_gc,
-            )),
+            exception: DOMException::new_inherited(message, "RTCError".into()),
```

Comment:
According to https://w3c.github.io/webrtc-pc/#dom-rtcerror-constructor we should pass "OperationError" instead of "RTCError".

Reply:
Fixed.